static/css/base/helpers-misc.less

```diff
@@ -4,10 +4,6 @@
 // helpers-common provides a more standard set for usage.
 @import (less) "less/font-families.less";
 
-.collapse {
```

Comment:
burn it with fire

Reply:
Are you sure this hasn't broken a bunch of things?Helpers misc is rreferenced in all legacy css.
Note books pages no longer loads this file since my changes...